docs/src/examples/components/Dialog/BestPractices/DialogBestPractices.tsx

```diff
@@ -1,7 +1,26 @@
 import * as React from 'react'
 import ComponentBestPractices from 'docs/src/components/ComponentBestPractices'
+import { link } from '../../../../utils/helpers'
+import { Text } from '@stardust-ui/react'
 
-const doList = ['Do use `trapFocus` prop to control focus trapping behavior.']
+const doList = [
+  'Do use `trapFocus` prop to control focus trapping behavior.',
+  <Text>
+    Recommended{' '}
```

Comment:
Please change to:
Do use `aria-describedby` prop to override or omit description if the dialog content is complex (contains actionable elements). Refer to  {link(
      'Aria Dialog pattern',
      'https://www.w3.org/TR/wai-aria-practices-1.1/examples/dialog-modal/dialog.html',
    )}{' '} for details.